frontend/src/components/HospitalInventoryList.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react'
+import {HospitalInventory, fetchHospitalInventory} from '../apis/infomed'
+import ListGroup from 'react-bootstrap/ListGroup'
+import Badge from 'react-bootstrap/Badge'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import { useParams } from 'react-router-dom'
+
+interface HospitalInventoryListParams {
+  hospitalId: string;
+}
+
+export const HospitalInventoryList =  ({hospitalId}: HospitalInventoryListParams) => {
+
+  const [hospitalInventoryList, setHospitalInventoryList] = useState<HospitalInventory[]>([]);
+
+  useEffect(() => {
+    (async () => {
+      const inventoryList = await fetchHospitalInventory(hospitalId);
+      setHospitalInventoryList(inventoryList);
+    })();
+  });
+  
+  const inventoryItems = hospitalInventoryList.map((item) => 
+    <ListGroup.Item key={item.resourceType}>
+      <Row>
+        <Col xs={6}>{item.resourceType}</Col>
+        <Col xs={6}><Badge pill variant="primary">{item.available}</Badge>{' '} </Col>
```

Comment:
There should be a couple of cases here: an item with just a total number and an item with both total and available number. I would probably say something like `{available} out of {total}` in this case. Also, if the unit of measurement is present, it should be here.